### PR TITLE
Feature/api/get video metadata

### PR DESCRIPTION
**Pull request by weaselix**

### CODE REVIEW

File: TranscodingDemo/Server/views/get_video_metadata_view.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    def get(self, request, resource_id, *args, **kwargs):
+        try:
+            resource: Resource = Resource.from_id(resource_id)
+            self.check_object_permissions(request, resource)
+            
+            video_info = VideoInfo.from_resource(resource)
+            serializer = VideoMetadataSerializer(video_info)
+
+            return JsonResponse(serializer.data, safe=False)
+        except exceptions.TranscodingServiceException as e:
```
Comment (bidzyyys) [inline TranscodingDemo/Server/views/get_video_metadata_view.py]: Can this method raise only `TranscodingServiceException`?

File: TranscodingDemo/Server/views/get_video_metadata_view.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            logger.warning("Failed attempt to query metadata for resource [id={}]. Error: {}".format(resource_id, e.message))
+            raise Http404()
```
Comment (bidzyyys) [inline TranscodingDemo/Server/views/get_video_metadata_view.py]: Just a note, I will change to JSONResponse with status 404.

File: TranscodingDemo/Server/tests/test_video_resource.py [hunk 3]
```diff
@@ -155,3 +157,82 @@ def test_validation_codec(self):
...
+    def test_get_metadata_not_authorized(self):
+        token = self.acquire_token()
+        wrong_token = self.acquire_token()
+        resource_id = self.upload_file(token, self.default_video)
+
+        response = self.get_request('/OnlineTranscoding/resource/{}/metadata/'.format(resource_id), wrong_token)
+        self.assertEqual(response.status_code, http.HTTPStatus.FORBIDDEN)
+
+
+    def test_steal_someone_metadata_not_authorized(self):
```
Comment (bidzyyys) [inline TranscodingDemo/Server/tests/test_video_resource.py]: Duplicates `test_get_metadata_not_authorized`

File: TranscodingDemo/Server/views/get_video_metadata_view.py (hunk 1 above)
Comment (nieznanysprawiciel) [reply TranscodingDemo/Server/views/get_video_metadata_view.py]: This is base class for most of our exceptions, so it should catch everything what was raised by our code.

File: TranscodingDemo/Server/views/get_video_metadata_view.py (hunk 2 above)
Comment (weaselix) [reply TranscodingDemo/Server/views/get_video_metadata_view.py]: you right.

File: TranscodingDemo/Server/tests/test_video_resource.py (hunk 3 above)
Comment (weaselix) [reply TranscodingDemo/Server/tests/test_video_resource.py]: Yes, I did it intentionally.

**Comment by weaselix** [comment]
> Why is this necessary

what?

**Pull request merged**